InfoScreen.c

```diff
@@ -42,14 +42,19 @@ InfoScreen* InfoScreen_done(InfoScreen* this) {
 void InfoScreen_drawTitled(InfoScreen* this, const char* fmt, ...) {
    va_list ap;
    va_start(ap, fmt);
+
+   char* title = xMalloc(COLS + 1);
+   int len = vsnprintf(title, COLS + 1, fmt, ap);
+   if (len > COLS) memset(&title[COLS - 3], '.', 3);
```

Comment:
Please avoid braceless if-statements.

Reply:
I am not sure if we have agreed on a brace-style for one-line branches. @natoscott @ginggs @fasterit 
But at least put the branch into a separate line.

Reply:
The original source had it on multiple lines. ;-)

Reply:
Good point, another topic for a future style guide.

Reply:
OK, braces added, though I'm fine with both, but having an agreed style guide is great. 👍